src/components/ListItem.js

```diff
@@ -42,8 +69,16 @@ const ListItem = ({ listItem, itemId }) => {
         onChange={() => checkItem()}
         disabled={isChecked}
         checked={isChecked}
+        aria-label="Mark this item as purchased"
```

Comment:
Love that you added aria labels even though that wasn't in the AC of your issue. 👍

Reply:
Great work guys! Everything works well here. Nicely done 👍😊